dvc/repo/metrics/show.py

```diff
@@ -96,13 +102,27 @@ def show(
         all_commits=all_commits,
     ):
         metrics = _collect_metrics(repo, targets, recursive)
+
+        if not metrics_found and metrics:
+            metrics_found = True
+
         vals = _read_metrics(repo, metrics, rev)
 
         if vals:
             res[rev] = vals
 
     if not res:
-        raise NoMetricsError()
+        if metrics_found:
+            msg = (
+                "Could not parse metrics files. Use `-v` option to see more "
```

Comment:
for consistency with the origina message
```suggestion
                "Could not parse metric files. Use `-v` option to see more "
```